.github/workflows/community-contributor.yml

```diff
@@ -0,0 +1,39 @@
+name: Add Community Label
+
+on:
+    pull_request_target:
+        types: [opened]
+    issues:
+        types: [opened]
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+permissions: {}
+
+jobs:
+    verify:
+        name: Verify and add label
+        runs-on: ubuntu-20.04
+        permissions:
+            contents: read
+            pull-requests: write
+            issues: write
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup Node.js
+              uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
+
+            - name: Install Octokit
+              run: npm --prefix .github/workflows/scripts install @octokit/action
+
+            - name: Install Actions Core
+              run: npm --prefix .github/workflows/scripts install @actions/core
```

Comment:
Should we create a `package.json` and put those two packages as dependencies?

Reply:
No because then you dev dependencies which you don't need on development scenarios (locally). You would also then need to run `npm install` which we don't need here.

Reply:
For the other actions, we put the devDependencies in the main `package.json` (https://github.com/woocommerce/woocommerce-blocks/blob/373f31156de59ea9de4d5ac89460323113aff535/package.json/#L107-L108). This is important to lock the version. Currently, with this approach, there is the risk that this automation breaks if those two deps have an update that introduces breaking changes.

Reply:
@gigitux - if we're worried about breaking changes when deps updates, we could lock the version like this https://github.com/woocommerce/woocommerce/blob/trunk/.github/actions/setup-woocommerce-monorepo/action.yml#L30-L49

This is what Woo core does. What do you think about this method?

Reply:
I would prefer to keep it in package.json as we do for other actions. Do you see some downside to doing it?

Reply:
The down sides I see is if you want to use different versions of the package for different workflows you wouldn't be able to do that unless you also install individual versions (is that possible) in package.json. And if you could, it would be confusing? Also you're now forcing local developers to install these packages when developing when they don't need these as it is only used by our GH actions workflow.

But I can also see the reason why you would suggest that. So if those reasons I mentioned above is not an issue for you, I am happy to put them in package.json.

Reply:
> The down sides I see is if you want to use different versions of the package for different workflows you wouldn't be able to do that unless you also install individual versions (is that possible) in package.json. And if you could, it would be confusing?

We should avoid using different versions of the package for different workflows. This approach could increase the maintenance cost.

> Also you're now forcing local developers to install these packages in their development environment when they don't need these as it is only used by our GH actions workflow.

Good point, but I don't see it so bad (at least for now). But I agree with you. Maybe could it make sense to add those deps as optional dependencies? Should we create an issue for this?

Reply:
@gigitux - Ok I have moved them to the package.json.